__tests__/app.test.js

```diff
@@ -275,4 +303,26 @@ describe("Error handling", () => {
         });
     });
   });
+  describe("8 GET /api/reviews (queries)", () => {
+    test("Responds with a 400 for invalid sort_query", () => {
+      return request(app)
+        .get("/api/reviews?sort_by=invalidSort")
+        .expect(400)
+        .then((response) => {
+          const {
+            body: { msg },
+          } = response;
+          expect(msg).toBe("Invalid sort query, try again.");
+        });
+    });
+    test("should return status: 400 for invalid order query", () => {
+      return request(app)
+        .get("/api/reviews?order=invalidOrder")
+        .expect(400)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toBe("Invalid order query, try again");
+        });
+    });
+  });
 });
```

Comment:
Please make sure you have tests for all of the the following queries  including topics - 
  - [ ] Status 200, accepts `order` query, e.g. `?order=desc`
  - [ ] Status 200, accepts `topic` query, e.g. `?topic=coding`
  - [ ] Status 404. non-existent `topic` query, e.g. `?topic=bananas`
  - [ ] Status 200. valid `topic` query, but has no articles responds with an empty array of articles, e.g. `?topic=paper`

Reply:
For topics? Is that meant as category? I cannot find topic anywhere in the db.